packages/auto-drive/src/api/models/uploads.ts

```diff
@@ -79,7 +80,7 @@ type ObjectUploadStatus<Type extends 'file' | 'folder'> =
       completed: true
       type: Type
       progress: number
-      cid: string
+      cid: AutoCID
```

Comment:
cid should remain a string, if you want to add an optional hash parameter that get return only if some optional input parameter is present, I'm fine with that.

But I don't want cid to become an object or to return the hash by default, I think this will be a step backward for the devEx,